File: dataset/Intermediate/Translation/Avatar/v2-Python-code/atcoder_ABC174_C.py
def calculate_t(val_two, t, val_one, k):
    return t % k * val_one + val_two
from sys import stdin, stdout, setrecursionlimit
from math import gcd, ceil, sqrt
from collections import Counter
from bisect import bisect_left, bisect_right
ii1 = lambda : int(stdin.readline().strip())
is1 = lambda : stdin.readline().strip()
iia = lambda : list(map(int, stdin.readline().strip().split()))
isa = lambda : stdin.readline().strip().split()
setrecursionlimit(100000)
mod = 1000000007
k = ii1()
t = 0
check_six = 399
check_one = 674
check_four = 970
check_two = 932
check_five = 389
check_three = 147
for iteration_count in range(1, k + 1):
    val_two = 7
    val_one = 10
    t = calculate_t(val_two, t, val_one, k)
    if check_five & check_three:
        if check_four & check_two:
            if check_six & check_one:
                if t % k == 0:
                    print(iteration_count)
                    break
else:
    print(-1)

File: scripts/GAs/Experiment_Results_v2-Python-code/intermediate/SR/Translation/use_test/deepseek-coder-33b-instruct/Avatar/Python/Java/atcoder_ABC174_C.java
```java
import java.util.*;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int k = scanner.nextInt();
        int t = 0;
        int check_six = 399;
        int check_one = 674;
        int check_four = 970;
        int check_two = 932;
        int check_five = 389;
        int check_three = 147;
        for (int iteration_count = 1; iteration_count <= k; iteration_count++) {
            int val_two = 7;
            int val_one = 10;
            t = calculateT(val_two, t, val_one, k);
            if ((check_five & check_three) != 0) {
                if ((check_four & check_two) != 0) {
                    if ((check_six & check_one) != 0) {
                        if (t % k == 0) {
                            System.out.println(iteration_count);
                            return;
                        }
                    }
                }
            }
        }
        System.out.println(-1);
    }

    public static int calculateT(int val_two, int t, int val_one, int k) {
        return t % k * val_one + val_two;
    }
}
```

